nvflare/private/fed/server/server_commands.py

```diff
@@ -174,16 +166,17 @@ def process(self, data: Shareable, fl_ctx: FLContext):
 
         """
 
-        shareable = data.get(ReservedKey.SHAREABLE)
-        shared_fl_ctx = data.get(ReservedKey.SHARED_FL_CONTEXT)
-        client = shareable.get_header(ServerCommandKey.FL_CLIENT)
-        fl_ctx.set_peer_context(shared_fl_ctx)
-        contribution_task_name = shareable.get_header(ServerCommandKey.TASK_NAME)
-        task_id = shareable.get_cookie(FLContextKey.TASK_ID)
         server_runner = fl_ctx.get_prop(FLContextKey.RUNNER)
-        server_runner.process_submission(client, contribution_task_name, task_id, shareable, fl_ctx)
+        if server_runner:
```

Comment:
What issue is this trying to address? We should not have this kind of scenario. If the server_runner has not been created, the client won't be able to get task to run and submit result. 
If the run already completed, the server parent process should not send request to here.